Cargo.toml

```diff
@@ -17,3 +16,5 @@ serde_json = "0.6.0"
 serde_macros = "0.6.11"
 uuid = "0.1.18"
 clippy = "0.0.37"
+docopt = "0.6.78"
+rustc-serialize = "0.3"
```

Comment:
add the new dependencies in alphabetical order (bonus points if you also move clippy)